### PR TITLE
makefile

### PR DESCRIPTION
**Pull request by aaaronhsu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 ## Usage
 1. Start Docker
+2. Run `make build run` to build and run the container.
+3. Run `make stop` to stop the container.
+
+## Usage (old but still works)
```
Comment (ChristianOkokhere) [inline README.md:8]: nit update header

**Review by nwang888 — APPROVED** [review]
lgtm

File: makefile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Makefile
+
```
Comment (ChristianOkokhere) [inline makefile:2]: Delete Space

**Review by ChristianOkokhere — COMMENTED** [review]
fix nits then merge :(

**Pull request merged**